mlflow/metrics/utils/prompts/v1.py

```diff
@@ -0,0 +1,94 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List
+
+from mlflow.metrics.base import (
+    EvaluationExample,
+)
+from mlflow.metrics.utils.prompt_template import (
+    PromptTemplate,
+)
+
+default_model = "openai:/gpt4"
+default_parameters = {
+    "temperature": 0.0,
+    "max_tokens": 100,
+    "top_p": 1.0,
```

Comment:
Should `top_p` and `temperature` both be specified? @suanmiao my understanding was that you should only set one of these

Reply:
I think we do plan to update this post testing. Will add a TODO to update this